packages/google_sign_in/google_sign_in_web/example/integration_test/gapi_load_test.dart

```diff
@@ -34,9 +34,12 @@ void main() {
     expect(() {
       plugin.initialized;
     }, throwsStateError,
-        reason:
-            'The plugin should throw if checking for `initialized` before calling .init');
-    await plugin.init(hostedDomain: '', clientId: '');
```

Comment:
The plugin still has an init method, don't remove the test for it (add a new one for .initWithParams)

Reply:
`gapi_load_legacy_init_test.dart` and `auth2_legacy_init_test.dart` contain the tests before migrating to `initWithParams`.

Reply:
Ahh, didn't notice those there, thanks!